File: URI Solutions/1008.cs
using System; 

class URI {

    static void Main(string[] args) { 

        int num, hour;
            double per_hour, salary;

            num = Convert.ToInt16(Console.ReadLine());
            hour = Convert.ToInt16(Console.ReadLine());
            per_hour = Convert.ToDouble(Console.ReadLine());

            salary = hour * per_hour;
            Console.WriteLine("NUMBER = " + num + "\nSALARY = U$ " + salary.ToString("0.00"));
    }

}

File: URI Solutions/1008.cpp
#include<iostream>
#include<iomanip>
using namespace std;

int main()
{
	int num,hour;
	float per_hour,salary;

	while(cin>>num>>hour>>per_hour)
        {
	  salary=hour*per_hour;
	  cout<<"NUMBER = "<<num<<endl<<"SALARY = U$ "<<fixed<<setprecision(2)<<salary<<endl;
	}
	
        return 0;
}
